docs/components/skeleton.md

```diff
@@ -132,3 +132,98 @@ This loading state is used for a spinner, which can be used for a wide variety o
   <KSkeleton type="spinner" />
 </template>
 ```
+
+## Full Screen Loading State
+The full screen loading state is used to display a full screen loader typically 
+during initial render of an app to avoid any FOUC while the app tries to figure
```

Comment:
you might want to say `FOUC (Flash of Unstyled Content` here just to save potential googling by people